### PR TITLE
Assert with error comments

### PR DESCRIPTION
**Pull request by egolol**
Comments for assert_wth_error class

### CODE REVIEW

**Review by annacanal — APPROVED** [review]

**Review by c-yuwei — APPROVED** [review]

File: braph2genesis/test/assert_with_error.m [hunk 1]
```diff
@@ -1,14 +1,22 @@
 function assert_with_error(code, error_identifier, varargin)
+%ASSERTWITHERROR checks if the error is correctly thrown by the code.
+%
+% ASSERTWITHERROR(CODE, ERRORIDENTIFIER, VARARGIN) checks if the correct
+%  error with ERRORIDENTIFIER is thrown by the CODE utilizing the format 
+%  given by the VARARGIN.
```
Comment (giovannivolpe) [inline braph2genesis/test/assert_with_error.m:6]: VARARGIN is not a format, but a series of variables.
in code they should be referred to as varargin{1}, varargin{2}, ... 
Please, explain it.

**Review by giovannivolpe — CHANGES_REQUESTED** [review]

File: braph2genesis/test/assert_with_error.m (hunk 1 above)
Comment (giovannivolpe) [reply braph2genesis/test/assert_with_error.m:6]: Also, add a base case without varargin as first case

**Review by annacanal — APPROVED** [review]

**Review by c-yuwei — APPROVED** [review]

File: braph2genesis/test/assert_with_error.m [hunk 2]
```diff
@@ -1,9 +1,13 @@
 function assert_with_error(code, error_identifier, varargin)
 %ASSERTWITHERROR checks if the error is correctly thrown by the code.
 %
+% ASSERTWITHERROR(CODE, ERRORIDENTIFIER) checks if the correct
+%  error with ERRORIDENTIFIER is thrown by the CODE.
+%
 % ASSERTWITHERROR(CODE, ERRORIDENTIFIER, VARARGIN) checks if the correct
-%  error with ERRORIDENTIFIER is thrown by the CODE utilizing the format 
-%  given by the VARARGIN. 
+%  error with ERRORIDENTIFIER is thrown by the CODE which uses the variables
```
Comment (c-yuwei) [inline braph2genesis/test/assert_with_error.m:8]: Maybe 'error' with capital E?

**Review by c-yuwei — APPROVED** [review]

File: braph2genesis/test/assert_with_error.m (hunk 2 above)
Comment (c-yuwei) [reply braph2genesis/test/assert_with_error.m:8]: no, it is not the beginning of the sentence.

**Review by giovannivolpe — COMMENTED** [review]
Seems done, let's double-check and merge it together when we speak

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**